{
  "uid" : "c0ab93634193d275",
  "name" : "Test submenu and button on the Capabilities page",
  "fullName" : "Test Redirection of Pages: Test submenu and button on the Capabilities page",
  "historyId" : "38e5c92b6f222b763bee5f9108c26853",
  "time" : {
    "start" : 1698734595409,
    "stop" : 1698734614721,
    "duration" : 19312
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"/html/body/div[1]/div[3]/div/div/div/div/main/article/div/section/div[21]/div/div/div/div/div/div[1]/div/div/div[2]/a\"}\n  (Session info: chrome=114.0.5735.133); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55df1abcc4e3 <unknown>\n#1 0x55df1a8fbc76 <unknown>\n#2 0x55df1a937c96 <unknown>\n#3 0x55df1a937dc1 <unknown>\n#4 0x55df1a9717f4 <unknown>\n#5 0x55df1a95703d <unknown>\n#6 0x55df1a96f30e <unknown>\n#7 0x55df1a956de3 <unknown>\n#8 0x55df1a92c2dd <unknown>\n#9 0x55df1a92d34e <unknown>\n#10 0x55df1ab8c3e4 <unknown>\n#11 0x55df1ab903d7 <unknown>\n#12 0x55df1ab9ab20 <unknown>\n#13 0x55df1ab91023 <unknown>\n#14 0x55df1ab5f1aa <unknown>\n#15 0x55df1abb56b8 <unknown>\n#16 0x55df1abb5847 <unknown>\n#17 0x55df1abc5243 <unknown>\n#18 0x7f4be8290402 <unknown>\n\n",
  "statusTrace" : "  File \"/usr/local/lib/python3.10/dist-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/usr/local/lib/python3.10/dist-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps/PageRedirectionSteps.py\", line 103, in step_impl\n    button_element = context.driver.find_element(By.XPATH, xpath)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"/html/body/div[1]/div[3]/div/div/div/div/main/article/div/section/div[21]/div/div/div/div/div/div[1]/div/div/div[2]/a\"}\n  (Session info: chrome=114.0.5735.133); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55df1abcc4e3 <unknown>\n#1 0x55df1a8fbc76 <unknown>\n#2 0x55df1a937c96 <unknown>\n#3 0x55df1a937dc1 <unknown>\n#4 0x55df1a9717f4 <unknown>\n#5 0x55df1a95703d <unknown>\n#6 0x55df1a96f30e <unknown>\n#7 0x55df1a956de3 <unknown>\n#8 0x55df1a92c2dd <unknown>\n#9 0x55df1a92d34e <unknown>\n#10 0x55df1ab8c3e4 <unknown>\n#11 0x55df1ab903d7 <unknown>\n#12 0x55df1ab9ab20 <unknown>\n#13 0x55df1ab91023 <unknown>\n#14 0x55df1ab5f1aa <unknown>\n#15 0x55df1abb56b8 <unknown>\n#16 0x55df1abb5847 <unknown>\n#17 0x55df1abc5243 <unknown>\n#18 0x7f4be8290402 <unknown>\n\n",
    "statusTrace" : "  File \"/usr/local/lib/python3.10/dist-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/usr/local/lib/python3.10/dist-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps/PageRedirectionSteps.py\", line 103, in step_impl\n    button_element = context.driver.find_element(By.XPATH, xpath)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Given The user is on the \"stg\" site",
      "time" : {
        "start" : 1698734596694,
        "stop" : 1698734605920,
        "duration" : 9226
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When The user hovers over the \"why-volt\" menu",
      "time" : {
        "start" : 1698734605920,
        "stop" : 1698734606242,
        "duration" : 322
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And The user clicks on the \"capabilites_4\" submenu link and verify the redirection \"capabilities_2\" page",
      "time" : {
        "start" : 1698734606242,
        "stop" : 1698734609210,
        "duration" : 2968
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When The user clicks on the \"top-10-capabilites\" button",
      "time" : {
        "start" : 1698734609210,
        "stop" : 1698734614651,
        "duration" : 5441
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"/html/body/div[1]/div[3]/div/div/div/div/main/article/div/section/div[21]/div/div/div/div/div/div[1]/div/div/div[2]/a\"}\n  (Session info: chrome=114.0.5735.133); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55df1abcc4e3 <unknown>\n#1 0x55df1a8fbc76 <unknown>\n#2 0x55df1a937c96 <unknown>\n#3 0x55df1a937dc1 <unknown>\n#4 0x55df1a9717f4 <unknown>\n#5 0x55df1a95703d <unknown>\n#6 0x55df1a96f30e <unknown>\n#7 0x55df1a956de3 <unknown>\n#8 0x55df1a92c2dd <unknown>\n#9 0x55df1a92d34e <unknown>\n#10 0x55df1ab8c3e4 <unknown>\n#11 0x55df1ab903d7 <unknown>\n#12 0x55df1ab9ab20 <unknown>\n#13 0x55df1ab91023 <unknown>\n#14 0x55df1ab5f1aa <unknown>\n#15 0x55df1abb56b8 <unknown>\n#16 0x55df1abb5847 <unknown>\n#17 0x55df1abc5243 <unknown>\n#18 0x7f4be8290402 <unknown>\n\n",
      "statusTrace" : "  File \"/usr/local/lib/python3.10/dist-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/usr/local/lib/python3.10/dist-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps/PageRedirectionSteps.py\", line 103, in step_impl\n    button_element = context.driver.find_element(By.XPATH, xpath)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "ac3c1f4a37c4e59c",
        "name" : "Error Message",
        "source" : "ac3c1f4a37c4e59c.txt",
        "type" : "text/plain",
        "size" : 960
      }, {
        "uid" : "dd1c8820bb51033e",
        "name" : "Screenshot of https://stg.voltactivedata.com/capabilities/ Page",
        "source" : "dd1c8820bb51033e.png",
        "type" : "image/png",
        "size" : 806684
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 2,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    }, {
      "name" : "Then The user should be redirected to the \"top-10-capabilites\" page",
      "time" : {
        "start" : 1698734614721,
        "stop" : 1698734614721,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When The user clicks on the \"avoid-it_2\" button",
      "time" : {
        "start" : 1698734614721,
        "stop" : 1698734614721,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Then The user should be redirected to the \"avoid-it\" page",
      "time" : {
        "start" : 1698734614721,
        "stop" : 1698734614721,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When The user clicks on the \"monitize_5g\" button",
      "time" : {
        "start" : 1698734614721,
        "stop" : 1698734614721,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Then The user should be redirected to the \"monitize_5g\" page",
      "time" : {
        "start" : 1698734614721,
        "stop" : 1698734614721,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 9,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "feature",
    "value" : "Test Redirection of Pages"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "c0ab93634193d275.json",
  "parameterValues" : [ ]
}